layer/empty.go

```diff
@@ -24,6 +26,13 @@ func (el *emptyLayer) TarStream() (io.ReadCloser, error) {
 	return ioutil.NopCloser(buf), nil
 }
 
+func (el *emptyLayer) TarSeekStream() (ioutils.ReadSeekCloser, error) {
+	buf := new(bytes.Buffer)
+	tarWriter := tar.NewWriter(buf)
+	tarWriter.Close()
```

Comment:
Here we create a TarWriter with `buf` as the place to write its data and immediately close it. This will cause `buf` to contain all the appropriate metadata of an empty tar archive. So we just do this to get a valid empty tar